builder/FindPackages.cmake

```diff
@@ -514,3 +514,16 @@ if( Linux )
   endif()
 endif()
 
+if (MFX_ENABLE_AENC)
+  find_library( AENC_LIBRARY libaenc.a PATHS ${CMAKE_HOME_DIRECTORY}/contrib/aenc/ )
+  if(NOT ${AENC_LIBRARY} MATCHES NOTFOUND)
```

Comment:
Looks it doesn't work this way. You made this check before defining MFX_ENABLE_AENC. Note absence "found here" message in the log.